app/controllers/menu_controller.rb

```diff
@@ -0,0 +1,5 @@
+class MenuController < ApplicationController
```

Comment:
what happened with the MenuItems controller?

Reply:
If you are going to use this controller for products, it needs to be named `ProductsController`

Reply:
Thanks! Done

Reply:
Done